xyz-psql-connector/src/main/java/com/here/xyz/psql/query/IterateChangesets.java

```diff
@@ -214,10 +214,7 @@ public ChangesetCollection handleChangesetCollection(ResultSet rs) throws SQLExc
         wroteStart = true;
       }
       
-      if (author == null) {
-          author = rs.getString("author");
-        }
-
+       author = rs.getString("author");
```

Comment:
This would set author of next version to the current change set

Reply:
@mujammil10  This issue in not happening. I have tested it and each changeset is getting assigned the proper author detail. I have also updated the unit test to validate this scenario